src/Network/Http.php

```diff
@@ -441,21 +442,32 @@ protected function headerToArray($header)
         return $headers;
     }
 
+    /**
+     * Add JSON encoded payload
+     */
+    public function json(array|string $payload): self
```

Comment:
json_encode can take a lot more types than just array|string, we should probably support everything that they do. To check to see if it needs to be encoded or not we can just pass it to Str::isJson()